build.sh

```diff
@@ -2,9 +2,9 @@
 
 cd "$(dirname "${0}")"
 
-# Support go 1 release 1.10 or higher
+# Support go 1 release 1.9 or higher
```

Comment:
curious why we're supporting an older version in this patch, is it intentional?

Reply:
Indeed, you are right to ask. There are two reasons, which I should have written a comment for. The first is I had a couple of strange intermittent crashes on Windows 7, where the go runtime actually crashes under go 1.10. I didn't experience this crash under go 1.9, so I thought it may be 1.10 related (although still under investigation). See for example [this task log](https://taskcluster-artifacts.net/S40V1BnRSd2NIADAQSGuvQ/0/public/logs/live_backing.log). It contains:

```
runtime: setevent failed; errno=6
fatal error: runtime.semawakeup

runtime stack:
runtime.throw(0x96bbae, 0x12)
	Z:/task_1519762930/go1.10/go/src/runtime/panic.go:619 +0x6b
runtime.semawakeup.func1()
	Z:/task_1519762930/go1.10/go/src/runtime/os_windows.go:599 +0x70
runtime.semawakeup(0x11d686c0)
	Z:/task_1519762930/go1.10/go/src/runtime/os_windows.go:597 +0x39
runtime.notewakeup(0x11d68774)
	Z:/task_1519762930/go1.10/go/src/runtime/lock_sema.go:147 +0x61
runtime.startm(0x0, 0x1)
	Z:/task_1519762930/go1.10/go/src/runtime/proc.go:2012 +0xb6
runtime.wakep()
	Z:/task_1519762930/go1.10/go/src/runtime/proc.go:2078 +0x4d
runtime.resetspinning()
	Z:/task_1519762930/go1.10/go/src/runtime/proc.go:2449 +0x7a
runtime.schedule()
	Z:/task_1519762930/go1.10/go/src/runtime/proc.go:2543 +0x100
runtime.park_m(0x1210d5e0)
	Z:/task_1519762930/go1.10/go/src/runtime/proc.go:2599 +0x9f
runtime.mcall(0x0)
	Z:/task_1519762930/go1.10/go/src/runtime/asm_386.s:406 +0x47

goroutine 1 [chan receive]:
testing.(*T).Run(0x11d1efc0, 0x9752c0, 0x1e, 0x9d91c4, 0x11d12001)
	Z:/task_1519762930/go1.10/go/src/testing/testing.go:825 +0x25a
testing.runTests.func1(0x11d1efc0)
	Z:/task_1519762930/go1.10/go/src/testing/testing.go:1063 +0x4b
testing.tRunner(0x11d1efc0, 0x11db1efc)
	Z:/task_1519762930/go1.10/go/src/testing/testing.go:777 +0xaa
testing.runTests(0x11f7db90, 0xcab980, 0x39, 0x39, 0x1f)
	Z:/task_1519762930/go1.10/go/src/testing/testing.go:1061 +0x21b
testing.(*M).Run(0x11f35080, 0x0)
	Z:/task_1519762930/go1.10/go/src/testing/testing.go:978 +0x136
main.main()
	_testmain.go:154 +0x11d

goroutine 18 [syscall]:
os/signal.signal_recv(0x0)
	Z:/task_1519762930/go1.10/go/src/runtime/sigqueue.go:139 +0x12f
os/signal.loop()
	Z:/task_1519762930/go1.10/go/src/os/signal/signal_unix.go:22 +0x1a
created by os/signal.init.0
	Z:/task_1519762930/go1.10/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 46 [runnable]:
io.(*pipe).Write(0x11e8e030, 0x11ee0180, 0x37, 0x40, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/pipe.go:89 +0x1b9
io.(*PipeWriter).Write(0x11e7e610, 0x11ee0180, 0x37, 0x40, 0x11ee0140, 0x37, 0x40)
	Z:/task_1519762930/go1.10/go/src/io/pipe.go:153 +0x3b
io.(*multiWriter).Write(0x11d7b1f0, 0x11ee0180, 0x37, 0x40, 0x37, 0x40, 0x1)
	Z:/task_1519762930/go1.10/go/src/io/multi.go:60 +0x6d
github.com/taskcluster/generic-worker.(*TaskRun).Log(0x11f9a000, 0x11e782a0, 0x27, 0x11dac1b0, 0x84)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/main.go:922 +0xf6
github.com/taskcluster/generic-worker.(*TaskRun).Info(0x11f9a000, 0x11dac1b0, 0x84)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/main.go:905 +0xa5
github.com/taskcluster/generic-worker.(*TaskRun).Infof(0x11f9a000, 0x96442d, 0x2, 0x11f997d0, 0x1, 0x1)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/main.go:892 +0x5d
github.com/taskcluster/generic-worker.(*TaskRun).ExecuteCommand(0x11f9a000, 0x0, 0x11f9a000)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/main.go:943 +0x2a1
github.com/taskcluster/generic-worker.(*TaskRun).Run(0x11f9a000, 0x1)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/main.go:1226 +0x162e
github.com/taskcluster/generic-worker.RunWorker(0x0)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/main.go:616 +0x501
github.com/taskcluster/generic-worker.execute(0x120342d0)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/helper_test.go:188 +0x8b
github.com/taskcluster/generic-worker.scheduleAndExecute(0x120342d0, 0x120a6460, 0x11d663c0, 0x6, 0x6, 0x11e54180, 0xe, 0x10, 0x0, 0x1, ...)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/helper_test.go:197 +0x52
github.com/taskcluster/generic-worker.TestProtectedArtifactsReplaced(0x120342d0)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/artifacts_test.go:490 +0x6f9
testing.tRunner(0x120342d0, 0x9d91c4)
	Z:/task_1519762930/go1.10/go/src/testing/testing.go:777 +0xaa
created by testing.(*T).Run
	Z:/task_1519762930/go1.10/go/src/testing/testing.go:824 +0x243

goroutine 97 [select]:
net/http.(*persistConn).writeLoop(0x1209a280)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1822 +0xfc
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1238 +0x815

goroutine 7 [IO wait]:
internal/poll.runtime_pollWait(0x32610f80, 0x72, 0x0)
	Z:/task_1519762930/go1.10/go/src/runtime/netpoll.go:173 +0x4b
internal/poll.(*pollDesc).wait(0x120ba0fc, 0x72, 0xc76100, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_poll_runtime.go:85 +0x8c
internal/poll.(*ioSrv).ExecIO(0xcc2ec8, 0x120ba014, 0x9d92cc, 0x1ff3, 0x11cf4790, 0xd)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:223 +0xf1
internal/poll.(*FD).Read(0x120ba000, 0x11e12000, 0x2000, 0x2000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:484 +0x1f0
net.(*netFD).Read(0x120ba000, 0x11e12000, 0x2000, 0x2000, 0xa, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/fd_windows.go:151 +0x3f
net.(*conn).Read(0x11cc4010, 0x11e12000, 0x2000, 0x2000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/net.go:176 +0x56
crypto/tls.(*block).readFromUntil(0x11cdc120, 0x32650000, 0x11cc4010, 0x5, 0x11cc4010, 0x11cf4678)
	Z:/task_1519762930/go1.10/go/src/crypto/tls/conn.go:493 +0x76
crypto/tls.(*Conn).readRecord(0x11cf46c0, 0x9d9717, 0x11cf4760, 0x1)
	Z:/task_1519762930/go1.10/go/src/crypto/tls/conn.go:595 +0xbd
crypto/tls.(*Conn).Read(0x11cf46c0, 0x120cf000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/crypto/tls/conn.go:1156 +0xdd
net/http.(*persistConn).Read(0x1209a000, 0x120cf000, 0x1000, 0x1000, 0x5, 0x1204df60, 0x11ec0930)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1453 +0x15b
bufio.(*Reader).fill(0x11d38120)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:100 +0xea
bufio.(*Reader).Peek(0x11d38120, 0x1, 0x0, 0x0, 0x0, 0x1206e300, 0x0)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:132 +0x2f
net/http.(*persistConn).readLoop(0x1209a000)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1601 +0x17c
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1237 +0x7f4

goroutine 8 [select]:
net/http.(*persistConn).writeLoop(0x1209a000)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1822 +0xfc
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1238 +0x815

goroutine 99 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x4b0, 0x1207a000, 0x8000, 0x11e0aec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x4b0, 0x1207a000, 0x8000, 0x8000, 0x11e0aec4, 0x0, 0x11e18580, 0x11e0aed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x4b0, 0x1207a000, 0x8000, 0x8000, 0x2f0d70, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e18580, 0x1207a000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e118, 0x1207a000, 0x8000, 0x8000, 0x11e0af54, 0x11e0af58, 0x11d10001)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e118, 0x1207a000, 0x8000, 0x8000, 0x8000, 0x8000, 0x11d10018)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e118, 0x1207a000, 0x8000, 0x8000, 0x2c, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e118, 0x2c, 0x0, 0x430f0e, 0x9d9660)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e118)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 70 [IO wait]:
internal/poll.runtime_pollWait(0x32610f00, 0x72, 0x0)
	Z:/task_1519762930/go1.10/go/src/runtime/netpoll.go:173 +0x4b
internal/poll.(*pollDesc).wait(0x11f6e0fc, 0x72, 0xc76100, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_poll_runtime.go:85 +0x8c
internal/poll.(*ioSrv).ExecIO(0xcc2ec8, 0x11f6e014, 0x9d92cc, 0x0, 0x0, 0x2)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:223 +0xf1
internal/poll.(*FD).Read(0x11f6e000, 0x120eb000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:484 +0x1f0
net.(*netFD).Read(0x11f6e000, 0x120eb000, 0x1000, 0x1000, 0x1, 0x0, 0x42bb2b)
	Z:/task_1519762930/go1.10/go/src/net/fd_windows.go:151 +0x3f
net.(*conn).Read(0x11e7e900, 0x120eb000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/net.go:176 +0x56
net/http.(*persistConn).Read(0x1209a140, 0x120eb000, 0x1000, 0x1000, 0x4, 0x11e7d0a0, 0x1206e870)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1453 +0x15b
bufio.(*Reader).fill(0x11e8e330)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:100 +0xea
bufio.(*Reader).Peek(0x11e8e330, 0x1, 0x0, 0x0, 0x0, 0x1206e880, 0x0)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:132 +0x2f
net/http.(*persistConn).readLoop(0x1209a140)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1601 +0x17c
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1237 +0x7f4

goroutine 71 [select]:
net/http.(*persistConn).writeLoop(0x1209a140)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1822 +0xfc
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1238 +0x815

goroutine 16 [IO wait]:
internal/poll.runtime_pollWait(0x32610e80, 0x72, 0x0)
	Z:/task_1519762930/go1.10/go/src/runtime/netpoll.go:173 +0x4b
internal/poll.(*pollDesc).wait(0x11f6e27c, 0x72, 0xc76100, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_poll_runtime.go:85 +0x8c
internal/poll.(*ioSrv).ExecIO(0xcc2ec8, 0x11f6e194, 0x9d92cc, 0xff3, 0x12108c10, 0xd)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:223 +0xf1
internal/poll.(*FD).Read(0x11f6e180, 0x11fb9000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:484 +0x1f0
net.(*netFD).Read(0x11f6e180, 0x11fb9000, 0x1000, 0x1000, 0xa, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/fd_windows.go:151 +0x3f
net.(*conn).Read(0x11e96010, 0x11fb9000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/net.go:176 +0x56
crypto/tls.(*block).readFromUntil(0x11e82160, 0x32650000, 0x11e96010, 0x5, 0x11e96010, 0x2c)
	Z:/task_1519762930/go1.10/go/src/crypto/tls/conn.go:493 +0x76
crypto/tls.(*Conn).readRecord(0x12108b40, 0x9d9717, 0x12108be0, 0x121d1f04)
	Z:/task_1519762930/go1.10/go/src/crypto/tls/conn.go:595 +0xbd
crypto/tls.(*Conn).Read(0x12108b40, 0x11fe5000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/crypto/tls/conn.go:1156 +0xdd
net/http.(*persistConn).Read(0x1209a280, 0x11fe5000, 0x1000, 0x1000, 0x2, 0x121d1f04, 0x2)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1453 +0x15b
bufio.(*Reader).fill(0x11e6b5c0)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:100 +0xea
bufio.(*Reader).Peek(0x11e6b5c0, 0x1, 0x0, 0x0, 0x0, 0x11cd4b40, 0x0)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:132 +0x2f
net/http.(*persistConn).readLoop(0x1209a280)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1601 +0x17c
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1237 +0x7f4

goroutine 100 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x4bc, 0x12100000, 0x8000, 0x1209fec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x4bc, 0x12100000, 0x8000, 0x8000, 0x1209fec4, 0x0, 0x11e18840, 0x1209fed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x4bc, 0x12100000, 0x8000, 0x8000, 0x2f035c, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e18840, 0x12100000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e168, 0x12100000, 0x8000, 0x8000, 0x2, 0x2, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e168, 0x12100000, 0x8000, 0x8000, 0x8000, 0x8000, 0x5bb1e7)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e168, 0x12100000, 0x8000, 0x8000, 0x1, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e168, 0x0, 0x0, 0x430f0e, 0x9d9660)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e168)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 101 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x4c8, 0x12070000, 0x8000, 0x120a3ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x4c8, 0x12070000, 0x8000, 0x8000, 0x120a3ec4, 0x0, 0x11e18b00, 0x120a3ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x4c8, 0x12070000, 0x8000, 0x8000, 0x2f0a14, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e18b00, 0x12070000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e1c0, 0x12070000, 0x8000, 0x8000, 0x1000000, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e1c0, 0x12070000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e1c0, 0x12070000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e1c0, 0x0, 0x0, 0x430f0e, 0x9d9660)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e1c0)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 113 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x494, 0x11fe6000, 0x8000, 0x120ffec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x494, 0x11fe6000, 0x8000, 0x8000, 0x120ffec4, 0x0, 0x11db0000, 0x120ffed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x494, 0x11fe6000, 0x8000, 0x8000, 0x44f470, 0x120ffed4, 0x40edd0)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db0000, 0x11fe6000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11d10f78, 0x11fe6000, 0x8000, 0x8000, 0x484dea, 0x11e19894, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11d10f78, 0x11fe6000, 0x8000, 0x8000, 0x8000, 0x8000, 0x1208c9a0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11d10f78, 0x11fe6000, 0x8000, 0x8000, 0x0, 0x12092730, 0x11e56588, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11d10f78, 0x1, 0x6e11f6, 0x12092700, 0x120fffdc)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11d10f78)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 80 [IO wait]:
internal/poll.runtime_pollWait(0x32610e00, 0x72, 0x0)
	Z:/task_1519762930/go1.10/go/src/runtime/netpoll.go:173 +0x4b
internal/poll.(*pollDesc).wait(0x11f6e6fc, 0x72, 0xc76100, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_poll_runtime.go:85 +0x8c
internal/poll.(*ioSrv).ExecIO(0xcc2ec8, 0x11f6e614, 0x9d92cc, 0x3f3, 0x12108550, 0xd)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:223 +0xf1
internal/poll.(*FD).Read(0x11f6e600, 0x120c9400, 0x400, 0x400, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:484 +0x1f0
net.(*netFD).Read(0x11f6e600, 0x120c9400, 0x400, 0x400, 0x0, 0xe, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/fd_windows.go:151 +0x3f
net.(*conn).Read(0x11e7fb60, 0x120c9400, 0x400, 0x400, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/net.go:176 +0x56
crypto/tls.(*block).readFromUntil(0x11f82de0, 0x32650000, 0x11e7fb60, 0x5, 0x11e7fb60, 0x2c)
	Z:/task_1519762930/go1.10/go/src/crypto/tls/conn.go:493 +0x76
crypto/tls.(*Conn).readRecord(0x12108480, 0x9d9717, 0x12108520, 0x1)
	Z:/task_1519762930/go1.10/go/src/crypto/tls/conn.go:595 +0xbd
crypto/tls.(*Conn).Read(0x12108480, 0x11fd7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/crypto/tls/conn.go:1156 +0xdd
net/http.(*persistConn).Read(0x1209a8c0, 0x11fd7000, 0x1000, 0x1000, 0x5, 0x11ed2700, 0x1206eb70)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1453 +0x15b
bufio.(*Reader).fill(0x11f02870)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:100 +0xea
bufio.(*Reader).Peek(0x11f02870, 0x1, 0x0, 0x0, 0x0, 0x12069240, 0x0)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:132 +0x2f
net/http.(*persistConn).readLoop(0x1209a8c0)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1601 +0x17c
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1237 +0x7f4

goroutine 81 [select]:
net/http.(*persistConn).writeLoop(0x1209a8c0)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1822 +0xfc
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1238 +0x815

goroutine 98 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x3f8, 0x11dd2000, 0x8000, 0x120b3ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x3f8, 0x11dd2000, 0x8000, 0x8000, 0x120b3ec4, 0x0, 0x11e18000, 0x120b3ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x3f8, 0x11dd2000, 0x8000, 0x8000, 0x2f06b8, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e18000, 0x11dd2000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e030, 0x11dd2000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e030, 0x11dd2000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e030, 0x11dd2000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e030, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e030)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 102 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x4d4, 0x11fda000, 0x8000, 0x11e07ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x4d4, 0x11fda000, 0x8000, 0x8000, 0x11e07ec4, 0x0, 0x11e18dc0, 0x11e07ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x4d4, 0x11fda000, 0x8000, 0x8000, 0x2f10cc, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e18dc0, 0x11fda000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e218, 0x11fda000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e218, 0x11fda000, 0x8000, 0x8000, 0x8000, 0x8000, 0x1206e900)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e218, 0x11fda000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e218, 0x0, 0x0, 0x10002, 0x3)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e218)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 103 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x4e0, 0x11e1a000, 0x8000, 0x11e0dec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x4e0, 0x11e1a000, 0x8000, 0x8000, 0x11e0dec4, 0x0, 0x11e19080, 0x11e0ded0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x4e0, 0x11e1a000, 0x8000, 0x8000, 0x2f1428, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e19080, 0x11e1a000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e288, 0x11e1a000, 0x8000, 0x8000, 0x65326932, 0x77414452, 0x48344f62)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e288, 0x11e1a000, 0x8000, 0x8000, 0x8000, 0x8000, 0x4d574b34)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e288, 0x11e1a000, 0x8000, 0x8000, 0xa2d2d2d, 0x4920230a, 0x65757373, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e288, 0x6365533d, 0x54657275, 0x74737572, 0x726f4320)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e288)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 104 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x4ec, 0x11da2000, 0x8000, 0x11e09ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x4ec, 0x11da2000, 0x8000, 0x8000, 0x11e09ec4, 0x0, 0x11e19340, 0x11e09ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x4ec, 0x11da2000, 0x8000, 0x8000, 0x2f1784, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e19340, 0x11da2000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e320, 0x11da2000, 0x8000, 0x8000, 0x2d455441, 0x2d2d2d2d, 0x20230a0a)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e320, 0x11da2000, 0x8000, 0x8000, 0x8000, 0x8000, 0x433d4e43)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e320, 0x11da2000, 0x8000, 0x8000, 0x20797261, 0xa224143, 0x65532023, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e320, 0x34313432, 0x36373836, 0x38383430, 0x35373233)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e320)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 105 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x4f8, 0x11e36000, 0x8000, 0x12005ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x4f8, 0x11e36000, 0x8000, 0x8000, 0x12005ec4, 0x0, 0x11e19600, 0x12005ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x4f8, 0x11e36000, 0x8000, 0x8000, 0x2f035c, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e19600, 0x11e36000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e370, 0x11e36000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e370, 0x11e36000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e370, 0x11e36000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e370, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e370)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 106 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x504, 0x11fee000, 0x8000, 0x120faec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x504, 0x11fee000, 0x8000, 0x8000, 0x120faec4, 0x0, 0x11e198c0, 0x120faed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x504, 0x11fee000, 0x8000, 0x8000, 0x2f1784, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e198c0, 0x11fee000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e3c0, 0x11fee000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e3c0, 0x11fee000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e3c0, 0x11fee000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e3c0, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e3c0)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 107 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x518, 0x12006000, 0x8000, 0x120fbec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x518, 0x12006000, 0x8000, 0x8000, 0x120fbec4, 0x0, 0x11e19b80, 0x120fbed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x518, 0x12006000, 0x8000, 0x8000, 0x2f1428, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11e19b80, 0x12006000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e410, 0x12006000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e410, 0x12006000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e410, 0x12006000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e410, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e410)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 108 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x52c, 0x11fc6000, 0x8000, 0x120feec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x52c, 0x11fc6000, 0x8000, 0x8000, 0x120feec4, 0x0, 0x1203e000, 0x120feed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x52c, 0x11fc6000, 0x8000, 0x8000, 0x2f10cc, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x1203e000, 0x11fc6000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e478, 0x11fc6000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e478, 0x11fc6000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e478, 0x11fc6000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e478, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e478)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 109 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x538, 0x11fce000, 0x8000, 0x12000ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x538, 0x11fce000, 0x8000, 0x8000, 0x12000ec4, 0x0, 0x1203e6e0, 0x12000ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x538, 0x11fce000, 0x8000, 0x8000, 0x2f035c, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x1203e6e0, 0x11fce000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e4f8, 0x11fce000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e4f8, 0x11fce000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e4f8, 0x11fce000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e4f8, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e4f8)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 110 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x544, 0x1210e000, 0x8000, 0x12001ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x544, 0x1210e000, 0x8000, 0x8000, 0x12001ec4, 0x0, 0x1203e9a0, 0x12001ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x544, 0x1210e000, 0x8000, 0x8000, 0x2f06b8, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x1203e9a0, 0x1210e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e7e548, 0x1210e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e7e548, 0x1210e000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e548, 0x1210e000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c8f0, 0x11e56878, 0xa2c8b0, 0x11e7e548, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c8f0, 0x11e56878, 0x11e7e548)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 111 [select]:
net/http.(*persistConn).roundTrip(0x1209a0a0, 0x11eec180, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:2033 +0x4e0
net/http.(*Transport).RoundTrip(0xcaaee0, 0x11e54280, 0xcaaee0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:422 +0x6ee
net/http.send(0x11e54280, 0xa2c5d0, 0xcaaee0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x402561, 0x2, ...)
	Z:/task_1519762930/go1.10/go/src/net/http/client.go:252 +0x151
net/http.(*Client).send(0x11cdc620, 0x11e54280, 0x0, 0x0, 0x0, 0x0, 0x0, 0x2, 0x11ccc2b4, 0x3a7a00, ...)
	Z:/task_1519762930/go1.10/go/src/net/http/client.go:176 +0xaa
net/http.(*Client).Do(0x11cdc620, 0x11e54280, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/http/client.go:615 +0x23c
github.com/taskcluster/generic-worker/livelog.(*LiveLog).connectInputStream.func1(0x11dc4180, 0x11cdc620, 0x11e54280)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/livelog/livelog.go:115 +0x3a
created by github.com/taskcluster/generic-worker/livelog.(*LiveLog).connectInputStream
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/livelog/livelog.go:109 +0x207

goroutine 118 [IO wait]:
internal/poll.runtime_pollWait(0x32610d80, 0x72, 0x0)
	Z:/task_1519762930/go1.10/go/src/runtime/netpoll.go:173 +0x4b
internal/poll.(*pollDesc).wait(0x11f6e3fc, 0x72, 0xc76100, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_poll_runtime.go:85 +0x8c
internal/poll.(*ioSrv).ExecIO(0xcc2ec8, 0x11f6e314, 0x9d92cc, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:223 +0xf1
internal/poll.(*FD).Read(0x11f6e300, 0x11fc0000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:484 +0x1f0
net.(*netFD).Read(0x11f6e300, 0x11fc0000, 0x1000, 0x1000, 0x358ae8, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/fd_windows.go:151 +0x3f
net.(*conn).Read(0x11e560a0, 0x11fc0000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/net.go:176 +0x56
net/http.(*persistConn).Read(0x1209a0a0, 0x11fc0000, 0x1000, 0x1000, 0x4, 0x11ffed30, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1453 +0x15b
bufio.(*Reader).fill(0x11eb6180)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:100 +0xea
bufio.(*Reader).Peek(0x11eb6180, 0x1, 0x11f00000, 0x11ffee04, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:132 +0x2f
net/http.(*persistConn).readLoop(0x1209a0a0)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1601 +0x17c
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1237 +0x7f4

goroutine 121 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x594, 0x12136000, 0x8000, 0x12125ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x594, 0x12136000, 0x8000, 0x8000, 0x12125ec4, 0x0, 0x11db0580, 0x12125ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x594, 0x12136000, 0x8000, 0x8000, 0x2f10cc, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db0580, 0x12136000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e56108, 0x12136000, 0x8000, 0x8000, 0x1000000, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e56108, 0x12136000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56108, 0x12136000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56108, 0x0, 0x0, 0x430f0e, 0x9d9660)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e56108)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 119 [runnable]:
internal/poll.(*FD).writeUnlock(0x11f6e300)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_mutex.go:246 +0x48
internal/poll.(*FD).Write(0x11f6e300, 0x11d04000, 0x3d, 0x1000, 0x3d, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:662 +0x10c
net.(*netFD).Write(0x11f6e300, 0x11d04000, 0x3d, 0x1000, 0x12054000, 0x8e35c0, 0x9406e0)
	Z:/task_1519762930/go1.10/go/src/net/fd_windows.go:163 +0x3f
net.(*conn).Write(0x11e560a0, 0x11d04000, 0x3d, 0x1000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/net.go:188 +0x56
net/http.persistConnWriter.Write(0x1209a0a0, 0x11d04000, 0x3d, 0x1000, 0x4599a0, 0x11eec380, 0x96454b)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1253 +0x3f
bufio.(*Writer).Flush(0x11eec380, 0x11e560e8, 0x96454b)
	Z:/task_1519762930/go1.10/go/src/bufio/bufio.go:573 +0x64
net/http/internal.(*chunkedWriter).Write(0x11e560f0, 0x12126000, 0x37, 0x8000, 0x37, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/http/internal/chunked.go:215 +0x161
io.copyBuffer(0x32611018, 0x11e560f0, 0xa2d830, 0x11d50140, 0x12126000, 0x8000, 0x8000, 0x8e3500, 0x8f0e80, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:402 +0x1f1
io.Copy(0x32611018, 0x11e560f0, 0xa2d830, 0x11d50140, 0x11e560f0, 0x11e560b0, 0x1, 0x1)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
net/http.(*transferWriter).WriteBody(0x11d50140, 0xa2b7f0, 0x11eec380, 0x2, 0x2)
	Z:/task_1519762930/go1.10/go/src/net/http/transfer.go:330 +0x11d
net/http.(*Request).write(0x11e54280, 0xa2b7f0, 0x11eec380, 0x0, 0x11eec3a0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/net/http/request.go:622 +0x5b4
net/http.(*persistConn).writeLoop(0x1209a0a0)
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1825 +0x17c
created by net/http.(*Transport).dialConn
	Z:/task_1519762930/go1.10/go/src/net/http/transport.go:1238 +0x815

goroutine 122 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x59c, 0x12146000, 0x8000, 0x11fffec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x59c, 0x12146000, 0x8000, 0x8000, 0x11fffec4, 0x0, 0x11db0840, 0x11fffed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x59c, 0x12146000, 0x8000, 0x8000, 0x2f1428, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db0840, 0x12146000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e56120, 0x12146000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e56120, 0x12146000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56120, 0x12146000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56120, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e56120)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 123 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5a8, 0x12156000, 0x8000, 0x12144ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5a8, 0x12156000, 0x8000, 0x8000, 0x12144ec4, 0x0, 0x11db0b00, 0x12144ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5a8, 0x12156000, 0x8000, 0x8000, 0x2f0a14, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db0b00, 0x12156000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e56140, 0x12156000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e56140, 0x12156000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56140, 0x12156000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56140, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e56140)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 124 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5a4, 0x1214e000, 0x8000, 0x12145ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5a4, 0x1214e000, 0x8000, 0x8000, 0x12145ec4, 0x0, 0x11db0dc0, 0x12145ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5a4, 0x1214e000, 0x8000, 0x8000, 0x2f0d70, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db0dc0, 0x1214e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e56160, 0x1214e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e56160, 0x1214e000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56160, 0x1214e000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56160, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e56160)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 125 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5b8, 0x1219e000, 0x8000, 0x1211eec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5b8, 0x1219e000, 0x8000, 0x8000, 0x1211eec4, 0x0, 0x11db1080, 0x1211eed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5b8, 0x1219e000, 0x8000, 0x8000, 0x2f0d70, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db1080, 0x1219e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e56188, 0x1219e000, 0x8000, 0x8000, 0xac051b3b, 0x2cf7d18a, 0xa541a7f)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e56188, 0x1219e000, 0x8000, 0x8000, 0x8000, 0x8000, 0xffd60862)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56188, 0x1219e000, 0x8000, 0x8000, 0x93a761e, 0x2211306e, 0xd7fa9a9c, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56188, 0x69682190, 0x1f0ddfac, 0x735ce13d, 0x578b9cda)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e56188)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 126 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5c4, 0x1217e000, 0x8000, 0x1211fec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5c4, 0x1217e000, 0x8000, 0x8000, 0x1211fec4, 0x0, 0x11db1340, 0x1211fed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5c4, 0x1217e000, 0x8000, 0x8000, 0x2f0a14, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db1340, 0x1217e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e561a8, 0x1217e000, 0x8000, 0x8000, 0x9aee10de, 0x15677166, 0xff51e1fd)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e561a8, 0x1217e000, 0x8000, 0x8000, 0x8000, 0x8000, 0xcdd1ed37)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e561a8, 0x1217e000, 0x8000, 0x8000, 0xa54de67d, 0x32b17a5f, 0xffedb925, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e561a8, 0x659a1d97, 0x3155b171, 0xefecbbae, 0xf56cdb43)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e561a8)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 127 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5c0, 0x12186000, 0x8000, 0x12140ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5c0, 0x12186000, 0x8000, 0x8000, 0x12140ec4, 0x0, 0x11db1600, 0x12140ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5c0, 0x12186000, 0x8000, 0x8000, 0x2f0000, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db1600, 0x12186000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e56270, 0x12186000, 0x8000, 0x8000, 0x0, 0x0, 0xd3c)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e56270, 0x12186000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56270, 0x12186000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56270, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e56270)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 128 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5d8, 0x12166000, 0x8000, 0x12141ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5d8, 0x12166000, 0x8000, 0x8000, 0x12141ec4, 0x0, 0x11db18c0, 0x12141ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5d8, 0x12166000, 0x8000, 0x8000, 0x2f0000, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db18c0, 0x12166000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e56280, 0x12166000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e56280, 0x12166000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56280, 0x12166000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56280, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e56280)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 161 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5d4, 0x1218e000, 0x8000, 0x12142ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5d4, 0x1218e000, 0x8000, 0x8000, 0x12142ec4, 0x0, 0x11db1b80, 0x12142ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5d4, 0x1218e000, 0x8000, 0x8000, 0x2f035c, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db1b80, 0x1218e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e56290, 0x1218e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e56290, 0x1218e000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56290, 0x1218e000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e56290, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e56290)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 162 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5ec, 0x1216e000, 0x8000, 0x12143ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5ec, 0x1216e000, 0x8000, 0x8000, 0x12143ec4, 0x0, 0x11db1e40, 0x12143ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5ec, 0x1216e000, 0x8000, 0x8000, 0x2f035c, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11db1e40, 0x1216e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e562a0, 0x1216e000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e562a0, 0x1216e000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e562a0, 0x1216e000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e562a0, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e562a0)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 163 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5f4, 0x12176000, 0x8000, 0x12164ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5f4, 0x12176000, 0x8000, 0x8000, 0x12164ec4, 0x0, 0x11d2cdc0, 0x12164ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5f4, 0x12176000, 0x8000, 0x8000, 0x2f06b8, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11d2cdc0, 0x12176000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e562b0, 0x12176000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e562b0, 0x12176000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e562b0, 0x12176000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e562b0, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e562b0)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 164 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x5e8, 0x12196000, 0x8000, 0x12165ec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x5e8, 0x12196000, 0x8000, 0x8000, 0x12165ec4, 0x0, 0x11d2d080, 0x12165ed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x5e8, 0x12196000, 0x8000, 0x8000, 0x2f06b8, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11d2d080, 0x12196000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e562c0, 0x12196000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e562c0, 0x12196000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e562c0, 0x12196000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e562c0, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e562c0)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf

goroutine 165 [syscall, locked to thread]:
syscall.Syscall6(0x771a96fb, 0x5, 0x600, 0x121a6000, 0x8000, 0x1213eec4, 0x0, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/runtime/syscall_windows.go:184 +0xcf
syscall.ReadFile(0x600, 0x121a6000, 0x8000, 0x8000, 0x1213eec4, 0x0, 0x11d2d340, 0x1213eed0)
	Z:/task_1519762930/go1.10/go/src/syscall/zsyscall_windows.go:312 +0x97
syscall.Read(0x600, 0x121a6000, 0x8000, 0x8000, 0x2f0a14, 0x40c085, 0x40ed9b)
	Z:/task_1519762930/go1.10/go/src/syscall/syscall_windows.go:295 +0x55
internal/poll.(*FD).Read(0x11d2d340, 0x121a6000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/internal/poll/fd_windows.go:476 +0x165
os.(*File).read(0x11e562d0, 0x121a6000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file_windows.go:204 +0x3d
os.(*File).Read(0x11e562d0, 0x121a6000, 0x8000, 0x8000, 0x8000, 0x8000, 0x0)
	Z:/task_1519762930/go1.10/go/src/os/file.go:107 +0x51
io.copyBuffer(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e562d0, 0x121a6000, 0x8000, 0x8000, 0x0, 0x0, 0x0, ...)
	Z:/task_1519762930/go1.10/go/src/io/io.go:400 +0x138
io.Copy(0xa2c430, 0x11d7b1f0, 0xa2c8b0, 0x11e562d0, 0x0, 0x0, 0x0, 0x0)
	Z:/task_1519762930/go1.10/go/src/io/io.go:362 +0x51
github.com/taskcluster/generic-worker/process.(*Command).DirectOutput.func1(0xa2c430, 0x11d7b1f0, 0x11e562d0)
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:206 +0x3b
created by github.com/taskcluster/generic-worker/process.(*Command).DirectOutput
	Z:/task_1519762930/gopath1.10/src/github.com/taskcluster/generic-worker/process/process_windows.go:205 +0xaf
```

I think the underlying problem might be a go routine writing to a closed pipe, but it presents itself as a go runtime crash, which might be a problem with the new go 1.10 release on Windows 7.

The second reason is much more mundane - travis doesn't yet support go 1.10 natively, and I didn't want to introduce code to manually install go 1.10 in the travis build, as it is quite ugly to do so (we used to install go during the build, but I recently removed it).

I'll add a comment to explain the reason for it.